#!/usr/bin/python3
def safe_print_integer_err(value):
    import sys
    e=0
    try:
        e=print('{:d}'.format(value))
        return(True)
    except Exception as e:
        print('Exception: {}'.format(str(e)))
        return(False)
    except:
        return(False)
